src/parsers/manifestjson.js

```diff
@@ -149,7 +168,10 @@ export default class ManifestJSONParser extends JSONParser {
 
     this.isValid = validate(this.parsedJSON);
     if (!this.isValid) {
-      log.debug('Schema Validation messages', validate.errors);
+      log.debug(
+        'Schema Validation messages',
+        JSON.stringify(validate.errors, null, 2)
```

Comment:
Not strictly related to this PR, but while working on it I noticed that the error were logged as the string '[Object]'... which is not very helpful as a debug log message :-)